src/main/steps/c100-rebuild/check-your-answers/helpers/miamHelper.ts

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { MiamNonAttendReason } from '../../../../app/case/definition';
+import { HTML } from '../common/htmlSelectors';
+
+class MiamHelperDataParser<T> {
+  [x: string]: T;
+}
+//eslint @typescript-eslint/no-explicit-any
+const InstanceOfMiamHelper = new MiamHelperDataParser<any>();
+
+type KeysType = {
```

Comment:
@DaimDN These are not related Miam, these are related to type of order selection.
Can we remove if these are not used in this ?